.gitignore

```diff
@@ -18,7 +18,9 @@ _cgo_gotypes.go
 _cgo_export.*
 
 _testmain.go
-carbon-clickhouse
+
+/carbon-clickhouse
+/e2e-test
```

Comment:
I added `/` in the beginning so that files in `cmd/e2e-tests` won't be ignored. And made the same for `carbon-clickhouse` for sake of consistency.